eng/devices/android.cake

```diff
@@ -344,7 +345,24 @@ Task("uitest")
 	SetEnvironmentVariable("APPIUM_LOG_FILE", $"{BINLOG_DIR}/appium_android.log");
 
 	Information("Run UITests project {0}", PROJECT.FullPath);
-	RunTestWithLocalDotNet(PROJECT.FullPath, CONFIGURATION,	noBuild: true, resultsFileNameWithoutExtension: $"{name}-{CONFIGURATION}-android");
+
+	var adb = $"{ANDROID_SDK_ROOT}/platform-tools/adb";
+	
+	StartProcess("bash", $"{adb} logcat -c");
+	StartProcess("bash", $"{adb} logcat  -G 4M");
```

Comment:
I see you switched from AdbShell but it is still being used in the old code. Is it not working there either?

Reply:
yea, just throwing various pieces of spaghetti at the wall.